globals.d.ts

```diff
@@ -0,0 +1,70 @@
+declare module "tiny-markdown-editor" {
+  type CursorPosition = { col: number; row: number; node: string };
+
+  type CommandName =
+    | "bold"
+    | "italic"
+    | "strikethrough"
+    | "code"
+    | "h1"
+    | "h2"
+    | "ul"
+    | "ol"
+    | "blockquote"
+    | "hr"
+    | "insertLink"
+    | "insertImage";
+
+  type ChangeEvent = { content: string; linesDirty: boolean[] };
+
+  type SelectionEvent = {
+    focus: CursorPosition;
+    anchor: boolean;
+    commandState: Record<string | CommandName, boolean>;
```

Comment:
This isn't quite correct; not all the entries of `CommandName` can occur as a key (`insertLink`, `insertImage`, and `hr` can't), and there are some others that can (eg. `h3`..`h6`). The docs aren't super clear on this. I will clarify in the docs, and also change the type declaration accordingly.